apps/challenges/views.py

```diff
@@ -816,6 +817,28 @@ def create_challenge_using_zip_file(request, challenge_host_team_pk):
             response_data = {"error": message}
             return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
 
+        # To ensure that the schema for submission metaattributes is valid.
+        if (data.get("submission_meta_attributes_schema")):
+            for attribute in data["submission_meta_attributes_schema"]:
+                keys = ["name", "description", "type"]
+                missing = keys_exists_in_json(attribute, keys)
+
+                if len(missing) == 0:
+                    valid_attribute_types = ["boolean", "text", "radio", "checkbox"]
+                    attribute_type = attribute["type"]
+                    if attribute_type in valid_attribute_types:
+                        if attribute_type == "radio" or attribute_type == "checkbox":
+                            options = attribute.get("options")
+                            if not options or not len(options):
+                                message = "Please include at least one option in attribute for challenge_phase {}".format(data["id"])
+                                response_data = {"error": message}
+                                return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
+                else:
+                    missing_string = ", ".join(missing)
+                    message = "Please enter the following to attribute in phase {}: {}.".format(data["id"], missing_string)
```

Comment:
message isn't clear should it be `Please enter the following to meta attributes in phase`? or `Please enter the following to submission meta attributes in phase`?

Reply:
Will make the change.